cli/command/image/build.go

```diff
@@ -115,6 +116,7 @@ func NewBuildCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.StringVar(&options.networkMode, "network", "default", "Set the networking mode for the RUN instructions during build")
 	flags.SetAnnotation("network", "version", []string{"1.25"})
 	flags.Var(&options.extraHosts, "add-host", "Add a custom host-to-IP mapping (host:ip)")
+	flags.StringVar(&options.target, "target", "", "Set the target build stage to build. Defaults to last.")
```

Comment:
"Exit after target build stage has completed" is specific to current implementation. When the implementation changes in a future release, it should be invisible to the user, some cases just become faster.

Do you want me to skip the hint about default. I added it so that it is clear that passing this value is optional.